src/main/java/com/swapnixi/feedbackhub/service/implementation/UserServiceImpl.java

```diff
@@ -64,22 +82,16 @@ public List<User> getAllUsers() {
     public void deleteUser(Long userId) {
         // Implementation for deleting a user by ID
         // This method should delete the user with the given ID from the repository
-        throw new UnsupportedOperationException("Unimplemented method 'deleteUser'");
+        if (userRepository.findById(userId).isPresent()) {
+            userRepository.deleteById(userId);
+        } else {
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND);
+        }
     }
 
     @Override
     public List<User> getUsersWithNameContainingNewKeyword() {
-        return  userRepository.nameHasNewKeyword();
-    }
-
-    private User DtoToUser(UserDTO userDTO) {
-        User user = new User();
-        user.setId(userDTO.getId());
-        user.setName(userDTO.getName());
-        user.setEmail(userDTO.getEmail());
-        user.setAbout(userDTO.getAbout());
-        user.setPassword(userDTO.getPassword());
-        return user;
+        return userRepository.nameHasNewKeyword();
     }
```

Comment:
I have not removed this function, I dont know why this is showing that I have removed this function.